XIVSlothCombo/ConfigWindow.cs

```diff
@@ -842,8 +846,12 @@ private static void DrawUserConfigs(CustomComboPreset preset, bool enabled)
             // ====================================================================================
             #region SAGE
 
-            if (preset == CustomComboPreset.SGE_ST_Dosis_EDosisHPPer)
-                ConfigWindowFunctions.DrawSliderInt(0, 100, SGE.Config.SGE_ST_Dosis_EDosisHPPer, "Enemy HP %% Threshold");
+            if (preset == CustomComboPreset.SGE_ST_Dosis_EDosisHPLimiters)
+            {
+                ConfigWindowFunctions.DrawSliderInt(0, 100, SGE.Config.SGE_ST_Dosis_EDosisHPPer, "Enemy HP % Threshold");
```

Comment:
Does `%` work here? I'd noticed that `%%` displays both but `%` displays as whitespace last I looked.

Reply:
So after experimenting, these are the rules
Feature/Option Name: Need %% but, it will show as %%. % will crash. It's an issue with ImGUI's Checkbox
Feature/Option Description: %% to render correctly because it can have it's color changed, etc. % is invisible
Sliders: % is fine. Description string is separate from the slider, and is sanitized in DrawSliderInt (ie can't have it's color changed)